example/opensourcecmf/opensource_replica_msgs.cmf

```diff
@@ -0,0 +1,62 @@
+# Definition of client-server interface for the opensource example test application.
```

Comment:
This is not a client-server interface. These are the messages/commands for the open source state machine example (effectively, a byzantine fault-tolerant K/V store).